paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetTopBar.kt

```diff
@@ -40,28 +39,28 @@ import com.stripe.android.ui.core.R as StripeUiCoreR
 
 @Composable
 internal fun PaymentSheetTopBar(
-    viewModel: BaseSheetViewModel,
+    screen: PaymentSheetScreen,
+    paymentMethods: List<PaymentMethod>?,
```

Comment:
I'm wondering if we can turn this into a boolean, looks like it could be `showEditMenu`.